Updates PMC Weapon Prices

BlueHNT opened the pull request:
<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->

## About the PR
<!-- What did you change? -->
<!-- If this is a code change, summarize at high level how your new code works. This makes it easier to review. -->
Simply said, the current prices are dirt cheap. The guns you buy are investments for the entire round and buying better ones shouldn't cost a pack of cigs.

Liberty Vendor sale prices are calculated via taking `StaticPrice`*12
## Why / Balance
<!-- Discuss how this would affect game balance or explain why it was changed. Link any relevant discussions or issues. -->
Guns aren't cheap.

## Media
<!-- Attach media if the PR makes ingame changes (clothing, items, features, etc). 
Small fixes/refactors are exempt. Media may be used in SS14 progress reports with credit. -->

![image](https://github.com/user-attachments/assets/73b6e488-2231-44cd-83bd-218b7787ab55)

## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes
<!-- List any breaking changes, including namespaces, public class/method/field changes, prototype renames; and provide instructions for fixing them.
This will be posted in #codebase-changes. -->

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->

:cl:
- tweak: PMC Weapon prices increased

BlueHNT committed:
updates merc liberty vend prices

Danchi299 commented:
@BlueHNT I ughh... had a sudden urge to continue coding for today and accidentally did the PR. Sorry.
![image](https://github.com/user-attachments/assets/219bfc1f-6082-4ae3-8956-ec916ea1ccff)
I also did it without touching upstream so that may be good,

Can you close this PR and assist me on mine instead?

BlueHNT commented:
> @BlueHNT I ughh... had a sudden urge to continue coding for today and accidentally did the PR. Sorry.

> Can you close this PR and assist me on mine instead?

I don't see a reason to undo these changes, since it's already finished PR with values that I checked in on with Red and some players. You've done some additional sorting and probably have something in mind as well I'm guessing?

Danchi299 commented:
> I don't see a reason to undo these changes, since it's already finished PR with values that I checked in on with Red and some players. You've done some additional sorting and probably have something in mind as well I'm guessing?

Yeah, I Sorted it by category then price, added more weapons, added extended description to PMC firearms, and adjusted the prices.
Also it doesn't touch frontier file folder so it will cause less problems with mergin upstream in the future.
if you say people agree on your prices I can copy those to my PR, or you can give me co-authorship on this PR and I can add my changes.

Or I can add my thing after this is merged, but that is probably gonna be in a day.

BlueHNT commented:
> > I don't see a reason to undo these changes, since it's already finished PR with values that I checked in on with Red and some players. You've done some additional sorting and probably have something in mind as well I'm guessing?
> 
> Yeah, I Sorted it by category then price, added more weapons, added extended description to PMC firearms, and adjusted the prices. Also it doesn't touch frontier file folder so it will cause less problems with mergin upstream in the future. if you say people agree on your prices I can copy those to my PR, or you can give me co-authorship on this PR and I can add my changes.
> 
> Or I can add my thing after this is merged, but that is probably gonna be in a day.

I reckon if you copy the prices and set up the new guns around it. It'll work.

Danchi299 commented:
Doing it.
